packages/api-svc/src/routes.ts

```diff
@@ -414,7 +437,8 @@ export class ExpressRoutes {
 					return;
 				}
 				const batchIds = matrix.batches.map(item => item.id);
-				settlementTransfers = await this._batchTransferRepo.getBatchTransfersByBatchIds(batchIds);
+				const result = await this._batchTransferRepo.getBatchTransfersByBatchIds(batchIds);
+				settlementTransfers = result.items;
```

Comment:
This method returns on some if branches an array of transfers and another pagination results object. Should return always the same structure.